README.md

```diff
@@ -1,2 +1,15 @@
 # react-native-oauth2
 OAuth2 Package for RN
+
+## Usage
+```javascript
+  import Oauth from 'react-native-oauth2'
+
+  const client = new Oauth(APP_ID, APP_CALLBACK, ACCOUNTS_URL, ACCOUNTS_URL)
```

Comment:
Use AUTH_URL and TOKEN_URL instead of ACCOUNTS_URL

Reply:
fixed.